wp-content/themes/reconasia/assets/_scss/components/_post-block.scss

```diff
@@ -1,22 +1,89 @@
 @use 'abstracts' as *;
 
 .post-block {
+  display: grid;
+  grid-template-areas:
+    'img'
+    'title'
+    'excerpt'
+    'authors'
+    'date'
+    'categories';
+  grid-template-columns: 1fr;
+  margin-top: rem(40);
+  margin-right: auto;
+  margin-left: auto;
+
+  &.has-post-thumbnail {
+    @include breakpoint('medium') {
+      grid-template-areas:
+        'title img'
+        'excerpt img'
+        'authors img'
+        'date img'
+        'categories img';
+      grid-template-columns: 2fr 1fr;
+      grid-template-rows: max-content max-content max-content max-content auto;
+      column-gap: rem(32);
+    }
+  }
+
   &__title {
-    color: $color-black-100;
+    grid-area: title;
+    color: $color-black-190;
+    @include font-size(20px);
```

Comment:
Instead of setting the font-size here (which will be overwritten when you extend the placeholder), we should use the `font-clamp` include on the `%text-style-heading-large-serif-bold` placeholder. You'll need to do a rebase from `main` to pull it in.